app/models/manageiq/providers/network_manager.rb

```diff
@@ -25,6 +25,21 @@ class << model_name
     has_many :load_balancer_health_checks,        :foreign_key => :ems_id, :dependent => :destroy
     has_many :load_balancer_health_check_members, :through => :load_balancer_health_checks
 
+    # Generates ORDER BY ((SELECT COUNT(*) FROM "vms" WHERE "ext_management_systems"."parent_ems_id" = "vms"."ems_id"))
```

Comment:
hm I think it need to be here, it's specific to NetworkManager (or StorageManager)